samples/openthread/cli/README.rst

```diff
@@ -454,14 +459,65 @@ To test the Thread 1.2 features, complete the following steps:
       .. code-block:: console
 
          uart:~$ ot ping fe80:0:0:0:acbd:53bf:1461:a861
-         uart:~$ W: TX_STARTED event will be triggered without delay
          16 bytes from fe80:0:0:0:acbd:53bf:1461:a861: icmp_seq=3 hlim=64 time=421ms
          1 packets transmitted, 1 packets received. Packet loss = 0.0%. Round-trip min/a
          Done
 
       Observe that the reply latency is reduced to a value below 500 ms.
       The reduction occurs because the transmission from the leader is performed using CSL, based on the CSL Information Elements sent by the CSL Receiver.
 
+#. Verify the Service Registration Protocol (SRP) functionality.
+
+   a. On the leader kit, enable the SRP Server function:
+
+      .. code-block:: console
+
+         uart:~$ ot srp server enable
+         Done
+
+   #. Register a `_ipps._tcp`` service on the router kit (now SED):
+
+      .. code-block:: console
+
+         uart:~$ ot srp client host name my-host
+         Done
+         uart:~$ ot srp client host address fdde:ad00:beef:0:e0fc:dc28:1d12:8c2
+         Done
+         uart:~$ ot srp client service add my-service _ipps._tcp 12345
+         Done
+         uart:~$ ot srp client autostart enable
+         Done
+
+   #. Check if the host and service has been successfully registered:
```

Comment:
```suggestion
   #. On the router kit (now SED), check that the host and service have been successfully registered:
```
Adding mention of the router kit is not 100 % required, but better to be extra thorough.